lib/osml/tile_server/testing/ts_test_runner.ts

```diff
@@ -150,5 +151,8 @@ export class TSTestRunner extends Construct {
       memorySize: 1024,
       functionName: "TSTestRunner"
     });
+    if (ecrDeployment) {
+      this.lambdaIntegRunner.node.addDependency(ecrDeployment);
+    }
```

Comment:
This should be done at the stack level rather than inside the cdk contructs.

Reply:
refactored cdk constructs